batch/Makefile

```diff
@@ -13,7 +13,7 @@ BATCH_TEST_IMAGE = gcr.io/$(PROJECT)/batch-test:$(shell docker images -q --no-tr
 
 PY_FILES = $(shell find batch -iname \*.py -not -exec git check-ignore -q {} \; -print)
 
-PYTHONPATH := $${PYTHONPATH:+$${PYTHONPATH}:}../hailjwt
+PYTHONPATH := ../hailjwt$${PYTHONPATH:+:$${PYTHONPATH}}
```

Comment:
Why is this necessary?

Reply:
just seems right. I don't want to pick up a hailjwt that was accidentally pip installed. This is all irrelevant anyway with build.yaml